docs/shell-tips.md

```diff
@@ -1,83 +1,64 @@
 # Shell tips
 
-The *shell* (sometimes called *terminal* or *console*), is a **command line
-interpreter**. This is how most shells look like:
+The *shell* a is a **command line interpreter**. To use it you can open a
+*terminal* (sometimes called a *console*). This is how most terminal windows
+look like:
 
 ![An example shell window](shell-screenshot.png)
 
 If you haven't used it before, you should probably take a look at
-[this tutorial](http://linuxcommand.org/lc3_learning_the_shell.php). If you're
+[this tutorial](http://linuxcommand.org/lc3_learning_the_shell.php).
+
+If you're
 using Windows,
 [these videos](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGDV6SnbINlVUd0o2xT4JbMu)
 may be useful too, but keep in mind that the following tips only apply to
-Linux/macOS environments (Unix-like shells).
+Linux/macOS environments (Unix shells). You can also use a Unix-like shell
```

Comment:
How come the -like part is removed? Linux is technical not under Unix, at least it is not Unix certified.